backend/app/views/spree/admin/stock_movements/index.html.erb

```diff
@@ -41,7 +41,7 @@
   <div class="col-9 no-objects-found">
     <%= render 'spree/admin/shared/no_objects_found',
                  resource: Spree::StockMovement,
-                 new_resource_url: new_object_url %>
+                 new_resource_url: nil %>
```

Comment:
We can remove this local entirely if we use `local_assigns` in the partial above.